openverse_catalog/dags/common/slack.py

```diff
@@ -253,6 +253,21 @@ def should_send_message(http_conn_id=SLACK_NOTIFICATIONS_CONN_ID):
     return environment == "prod" or force_message
 
 
+def send_alert(
+    text: str,
+    username: str = "Airflow",
```

Comment:
Maybe a bit more specific here? Not sure if that really benefits us here, but thought I'd throw it out there :woman_shrugging: 
```suggestion
    username: str = "Airflow Alert",
```